frontend/android/app/src/main/AndroidManifest.xml

```diff
@@ -25,15 +25,6 @@
               android:name="io.flutter.embedding.android.NormalTheme"
               android:resource="@style/NormalTheme"
               />
-            <!-- Displays an Android View that continues showing the launch screen
-                 Drawable until Flutter paints its first frame, then this splash
-                 screen fades out. A splash screen is useful to avoid any visual
-                 gap between the end of Android's launch screen and the painting of
-                 Flutter's first frame. -->
-            <meta-data
-              android:name="io.flutter.embedding.android.SplashScreenDrawable"
-              android:resource="@drawable/launch_background"
-              />
```

Comment:
How did you come up with this change? Is this like in RN where there is a diff which should be applied to Flutter projects upon upgrading?

Reply:
And I think there was a deprecation notice in the console when building the app